src/app/modules/item/components/item-children-edit/item-children-edit.component.ts

```diff
@@ -139,14 +139,12 @@ export class ItemChildrenEditComponent implements OnChanges {
       ...child,
       scoreWeight: DEFAULT_SCORE_WEIGHT,
       isVisible: true,
-      ...(!child.id ? {
-        contentViewPropagation: 'as_info',
-        permissions: {
-          canView: 'solution',
-          canEdit: 'all_with_grant',
-          canGrantView: 'solution_with_grant',
-        },
-      } : {})
+      contentViewPropagation: child?.permissions?.canGrantView === 'none' ? 'none' : 'as_info',
```

Comment:
I assume we rely here that if it is a new child, it will be undefined, so `!== none`.
I am feel that it is a bit error prone, the next developer changing that may introduce an error.
I prefer you create a constant `permissionsForCreatedItem` (with the value you have on line 143-177), then use this constant when you do not have permissions.

BTW, `child?.pe...` -> child is not optional so I don't see the point of the optional chaining.

Reply:
Fixed